test/test_entanglement_tracker.jl

```diff
@@ -75,27 +75,29 @@ using Logging
 logger = ConsoleLogger(Logging.Debug; meta_formatter=(args...)->(:black,"",""))
 global_logger(logger)
```

Comment:
comment these out so that we can easily access them if we are debugging, but they are not in the way when just having normal tests -- this would create MB of data in the logs otherwise